mscore/mixer.cpp

```diff
@@ -125,6 +126,24 @@ Mixer::Mixer(QWidget* parent)
       retranslate(true);
       }
 
+
+
+//---------------------------------------------------------
+//   showDetailsToggled
+//---------------------------------------------------------
+
+void Mixer::toggleMuteChannels(){
+
+    for (int i =0; i < this->trackList.length(); i++){
+        if(this->trackList.at(i)->mti()->isMute())
```

Comment:
Seems this could be all condensed into two simple lines
`bool isMute = trackList.at(i)->mti()->isMute();`
`trackList.at(i)->mti()->setMute(isMute);`

Or possibly creating a simple function that toggles the mute in whatever `trackList.at(i)->mti()` returns.

Reply:
simplified in a newer change